### PR TITLE
RTP10b

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

**Comment by mattheworiordan** [comment]
LGTM

**Comment by tcard** [comment]
PTAL. The test wasn't actually working because we were simulating an ERROR protocol message that had nothing to do with the LEAVE operation, so it didn't affect it at all; the ACK arrived just fine. What we need is to replace the arriving ACK with a NACK to trigger the error in the callback.

**Comment by ricardopereira** [comment]
Cool, thanks.

File: Spec/RealtimeClientPresence.swift [hunk 1]
```diff
@@ -868,7 +868,7 @@ class RealtimeClientPresence: QuickSpec {
                     defer { client.close() }
                     let channel = client.channels.get("test")
 
-                    channel.attach()
+                                        channel.attach()
```
Comment (mattheworiordan) [inline Spec/RealtimeClientPresence.swift]: Odd spacing here?

Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: Done 3200787.

**Pull request merged**